addon/components/visualizer-course-session-types.js

```diff
@@ -18,6 +18,22 @@ export default class VisualizerCourseSessionTypes extends Component {
     return this.args.chartType || 'horz-bar';
   }
 
+  get filteredData() {
+    if (!this.data) {
+      return [];
+    }
+
+    let data = this.data;
+    if (this.args.filter) {
+      const exp = new RegExp(this.args.filter, 'gi');
```

Comment:
Late breaking comment... this probably needs `import { cleanQuery } from 'ilios-common/utils/query-utils';` to ensure the query string is regex safe before using it.

Reply:
good catch. fixed.